crux-llvm/tests/abduction/README.md

```diff
@@ -0,0 +1,52 @@
+### The Goal 
+What4 has a `get-abduct` command using cvc5, that given a goal that is disproved by the SMT solver (negation satisfiable), returns an abduct - a formula that entails the goal. We want to integrate abduction with online solving in crux. As a first attempt, we do the following. From the proof annotations (and the implicit goals) in a C file, crux creates a proof goal tree, a tree with a proof goal in each node. It traverses this tree and queries the SMT solver with the goal. If the query is unsatisfiable, the goal is successfully proved, if its satisfiable, the solver's counter-example is interpreted and output. We ask for and print one abduct in addition to the counter-example.
+
+### Test Files
+We have 9 test files over C ints, all unprovable:
+1. `abdpaper` the running example from the cvc5 abduction paper: does `y > 0 |= x + y + z > 0`? Acceptable abduct: `x + z > 0`.
+2. `addident`: does `|= x + y == x`? Acceptable abduct: `y = 0`.
+3. `addinv`: does `|= x + y == x`? Acceptable abduct: `y = -x`.
+4. `andex`: does `|= x & y == 1`? Acceptable abduct: `x = 1 ^ y = 1`.
+5. `file` from a crux-llvm example: does `x < 100 |= x + 1 < 100`? Acceptable abduct: `x < 99`.
+6. `maxint`: does `|= x + 1 > x`? Acceptable abduct: `x < maxint`.
+7. `multident`: does `|= x * y == x`? Acceptable abduct: `y = 1`.
+8. `multinv`: does `|= x * y == 0`? Acceptable abduct: `y = 0`.
+9. `trans`: does `x > y |= x > z`? Acceptable abduct: `y > z`.
+
+### Test Results
+With cvc5-1.0.2:
+| Test       | Entailment                | Baseline        | 8-bit Abducts                 | Time             | 32-bit Abducts          | Time           | Notes                                        |
+|------------|---------------------------|-----------------|-------------------------------|------------------|-------------------------|----------------|----------------------------------------------|
+| abdpaper   | `y > 0 \|= x + y + z > 0` | `x + z > 0`     | `(= (bvsrem z x) x)`          | real	3m0.023s  | `(= 0 z)`               | real	3m0.003s |                                              |
```

Comment:
Hm, it does look that way, but it seems to be rendered correctly, at least [here](https://github.com/GaloisInc/crucible/blob/crux-abduct/crux-llvm/tests/abduction/README.md). Could you check again and confirm is something needs to be changed?

Reply:
It doesn't need to hold up merging, but is there a reason to *not* change it?

Reply:
Sorry, what I mean to say is: It seems preferable to me that the headers are aligned, it makes the table a bit easier to read in a text editor. It also seems like a very low-cost fix. Together, these things make me think we should go ahead and fix it. Is there something I'm missing? Are you concerned that the table will no longer render correctly if the headers *are* aligned?

Reply:
I should have been clearer: the source-code seems to make it look like the headers are misaligned, but since the .md file is rendered as expected, it makes me think that the headers are actually aligned correctly. I'm realizing that I probably don't understand what you mean when you say they are misaligned. Sorry, I might not be seeing something very obvious - could you expand?

Reply:
Yeah, after checking this out locally it looks like this is just an artifact of Github's rendering. Weird!